noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/note.nr

```diff
@@ -152,13 +59,37 @@ pub fn compute_log<Note, let N: u32>(
 where
     Note: NoteType + Packable<N>,
 {
-    let final_plaintext = compute_note_plaintext_for_this_strategy(note, storage_slot);
+    let plaintext = compute_note_plaintext_for_this_strategy(note, storage_slot);
+
+    let ciphertext = AES128::encrypt_log(plaintext, recipient);
+
+    let log = prefix_with_tag(ciphertext, sender, recipient);
+
+    log
+}
+
+fn prefix_with_tag<let L: u32>(
```

Comment:
Wouldn't hurt to write a 1 line test for this, perhaps with mocked oracles.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/12942/commits/ffb9962445540ca198f5e7bf8e52f2040b8ceba6